Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/EyeTrackingTarget.cs

```diff
@@ -3,69 +3,186 @@
 
 using UnityEngine;
 using UnityEngine.Events;
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
 
-namespace Microsoft.MixedReality.Toolkit.Input
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
 {
     /// <summary>
-    /// A game object with the "Target" script attached can be selected and may react to hover states.
+    /// A game object with the "EyeTrackingTarget" script attached reacts to being looked at independent of other available inputs.
     /// </summary>
-    public class EyeTrackingTarget : BaseEyeFocusHandler, IMixedRealityPointerHandler, IMixedRealitySpeechHandler
+    [RequireComponent(typeof(EyeTrackingTarget))]
+    public class EyeTrackingTarget : InputSystemGlobalListener, IMixedRealityPointerHandler, IMixedRealitySpeechHandler
     {
         [SerializeField]
         private MixedRealityInputAction selectAction = MixedRealityInputAction.None;
 
         [SerializeField]
-        private MixedRealityInputAction[] voice_select = null;
+        private MixedRealityInputAction[] voiceSelect = null;
```

Comment:
We may also want to use FormerlySerializedAs here as well.

Reply:
(Actually for all of the rename/changes below as well)

Reply:
> (Actually for all of the rename/changes below as well)

Which other ones?